hack/verify-flags/exceptions.txt

```diff
@@ -5,11 +5,13 @@ cluster/aws/templates/configure-vm-aws.sh:  # We set the hostname_override to th
 cluster/aws/templates/configure-vm-aws.sh:  api_servers: '${API_SERVERS}'
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "hostname_override"
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "runtime_config"
+cluster/aws/templates/configure-vm-aws.sh:  kubelet_api_servers: '${KUBELET_APISERVER}'
```

Comment:
I believe flag verification does normalization by converting all dashes ('-') to underscores ('_'). Here's an example: in hack/verify-flags/known-flags.txt, `api-servers` (with a dash) is defined as a known flag. In hack/verify-flags/exceptions.txt, the following lines are defined:

```
cluster/juju/charms/trusty/kubernetes/hooks/api-relation-changed:        api_servers = 'http://%s:%s' % (api_info[0], api_info[1])
cluster/juju/charms/trusty/kubernetes/hooks/api-relation-changed:    api_servers = get_rel_hosts('api', rels, ('hostname', 'port'))
```

Notice that in both cases "api_servers" is with underscores. The same applies to the flag we're introducing here. If I misunderstood how this works, please let me know and I can correct it.